cdba.h

```diff
@@ -23,6 +23,8 @@ enum {
 	MSG_FASTBOOT_PRESENT,
 	MSG_FASTBOOT_DOWNLOAD,
 	MSG_FASTBOOT_BOOT,
+	MSG_FASTBOOT_BTN_PRESS,
+	MSG_FASTBOOT_BTN_RELEASE,
```

Comment:
protocol abi break

Reply:
I don't like the idea of having two messages for a single button. Maybe we should follow something like (MSG_BUTTON <button:u8, state:bool>)?

Reply:
+1 ^